views/layout.dt

```diff
@@ -1,12 +1,16 @@
 ﻿!!! 5
 html
+	- string resource(string name)
+		- import vibe.data.json;
+		- import std.file : readText;
+		- auto j = "public/manifest.json".readText.parseJsonString;
+		- return j[name].type == Json.Type.undefined ? "" : j[name].get!string;
```

Comment:
Hmm not so sure whether you want to read the JSON file on every request. A common strategy is to put the build in a unique directory and expose that one as environment variable. Of course, if the variable isn't defined, you could parse the manifest.
I know that with this approach it's harder to switch to new js without restarting the server, but afaict they are currently deployed together anyways​..
Alternatively if that's easier, just read the JSON once on startup.